jenkins/ci.suse.de/ardana-job.yaml

```diff
@@ -187,14 +194,24 @@
                                        -e "deployer_model=${model}" \
                                        -e "verification_temp_dir=$verification_temp_dir" \
                                        init.yml
+
+          run_playbook_on_deployer () {
+              ssh $sshargs ardana@$DEPLOYER_IP \
+                  "cd ~/scratch/ansible/next/ardana/ansible ;
+                   ansible-playbook -vvv -i hosts/verb_hosts $1"
+          }
+
           # Run site.yml outside ansible for output streaming
-          ssh $sshargs ardana@$DEPLOYER_IP "cd ~/scratch/ansible/next/ardana/ansible ; \
-               ansible-playbook -vvv -i hosts/verb_hosts site.yml"
-          if [ $? != 0 ]; then
+          if ! run_playbook_on_deployer site.yml; then
               echo "site.yml failed!  Aborting immediately." >&2
               exit $?
           fi
 
+          if ! run_playbook_on_deployer nova-compute-reboot.yml; then
```

Comment:
so I was assuming that we trigger a hard reboot (e.g. ssh reboot ) to simulate the "does the machine come up fine" case.. any reason you took the playbook here? does the playbook do more?